.travis.yml

```diff
@@ -3,17 +3,17 @@ sudo: required
 env:
     matrix:
         # openssl tests
-        - COMP=openssl OS_TYPE=centos OS_VERSION=5
+        #- COMP=openssl OS_TYPE=centos OS_VERSION=5
         - COMP=openssl OS_TYPE=centos OS_VERSION=6
         - COMP=openssl OS_TYPE=centos OS_VERSION=7
         - COMP=openssl OS_TYPE=fedora OS_VERSION=24
```

Comment:
`latest` sounds like a good choice considering fast release cycle of Fedora. But we should consider running the tests on other supported Fedora versions, which would require bumping anyway..